Backend/app/dashboard/Lib/src/Storage/Tabular/RegistrationToken.hs

```diff
@@ -36,6 +37,7 @@ mkPersist
       personId PersonTId
       merchantId MerchantTId
       createdAt UTCTime
+      operatingCity City
```

Comment:
We need to backfill all the data. Its better to make it Maybe, and handle like handled in BAP and BPP.

Reply:
In dashboard we don't have Cached Queries.
So wouldn't that need 2 DB queries.

Reply:
It is not for CachedQueries, what about DB data?

Reply:
Added migrations, as discussed.